pkg/services/alerting/notifiers/slack.go

```diff
@@ -73,25 +75,36 @@ func init() {
         </info-popover>
       </div>
       <div class="gf-form max-width-30">
-        <span class="gf-form-label width-6">Mention</span>
+        <span class="gf-form-label width-8">Mention User</span>
         <input type="text"
           class="gf-form-input max-width-30"
-          ng-model="ctrl.model.settings.mention"
+          ng-model="ctrl.model.settings.mentionUser"
           data-placement="right">
         </input>
         <info-popover mode="right-absolute">
-          Mention a user or a group using @ when notifying in a channel
+          Mention a user when notifying in a channel
         </info-popover>
       </div>
       <div class="gf-form max-width-30">
-        <span class="gf-form-label width-6">Token</span>
+        <span class="gf-form-label width-8">Mention Group</span>
+        <input type="text"
+          class="gf-form-input max-width-30"
+          ng-model="ctrl.model.settings.mentionGroup"
+          data-placement="right">
+        </input>
+        <info-popover mode="right-absolute">
+          Mention a group when notifying in a channel
```

Comment:
Need to extend this I think to include that you must reference user id's and how to include multiple

Reply:
@marefr Thanks, slipped my mind.